Broken grpc.WithBalancerName

After updating to `google.golang.org/grpc v1.46.0` I cannot compile anymore with the following error:
```
../../../../pkg/mod/github.com/weaveworks/common@v0.0.0-20210913144402-035033b78a78/httpgrpc/server/server.go:137:8: undefined: grpc.WithBalancerName
```

It seems that the `WithBalancerName` was actually removed in grpc after it was deprecated for some time https://github.com/grpc/grpc-go/pull/5232

The code in this repository that needs updating can be found here:
https://github.com/weaveworks/common/blob/f83ccc76d8237acd9d0fa0c68b61298fbb2cb03d/httpgrpc/server/server.go#L137

--- a/httpgrpc/server/server.go
+++ b/httpgrpc/server/server.go
@@ -17,7 +17,6 @@ import (
 	"github.com/sercand/kuberesolver"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/balancer/roundrobin"
 
 	"github.com/weaveworks/common/httpgrpc"
 	"github.com/weaveworks/common/logging"
@@ -132,9 +131,10 @@ func NewClient(address string) (*Client, error) {
 	if err != nil {
 		return nil, err
 	}
+	const grpcServiceConfig = `{"loadBalancingPolicy":"round_robin"}`
 
 	dialOptions := []grpc.DialOption{
-		grpc.WithBalancerName(roundrobin.Name),
+		grpc.WithDefaultServiceConfig(grpcServiceConfig),
 		grpc.WithInsecure(),
 		grpc.WithUnaryInterceptor(grpc_middleware.ChainUnaryClient(
 			otgrpc.OpenTracingClientInterceptor(opentracing.GlobalTracer()),

